packages/ag-charts-community/src/chart/axis/axis.ts

```diff
@@ -711,8 +711,6 @@ export abstract class Axis<S extends Scale<D, number, TickInterval<S>> = Scale<a
             return;
         }
 
-        this.setTickCount();
```

Comment:
This call doesn't do anything as it short circuit returns if no count param or not continuous (skipped on line 711).

Reply:
Nice spot!